### DISCUSSION TITLE
How can i open neorg files unfolded?

### DISCUSSION THREAD

**Discussion by otavioschwanck** [nvim-neorg/neorg#986] [General]
How can i open neorg files unfolded? i have to ever remove the folds to manage my file.

**Comment by NTBBloodbath** [comment]
You could write an autocommand that runs on `*.norg` files and executes the `zR` command. Something like this (did not test it, just a proof of concept and I do not know if it works!)

```lua
vim.api.nvim_create_autocmd("BufEnter", {
  pattern = "*.norg",
  command = "zR"
})
```

**Reply by otavioschwanck** [reply]
```lua
  vim.api.nvim_create_autocmd("BufRead", {
    pattern = "*.norg",
    command = "norm zR"
  })

```

did the trick, thanks for the suggestion!

**Comment by otavioschwanck** [comment]
> ```lua
> vim.api.nvim_create_autocmd("BufEnter", {
>   pattern = "*.norg",
>   command = "zR"
> })
> ```
